src/components/WalletModal/PendingView.tsx

```diff
@@ -105,31 +116,7 @@ export default function PendingView({
           )}
         </LoadingWrapper>
       </LoadingMessage>
-      {Object.keys(SUPPORTED_WALLETS).map(key => {
-        const option = SUPPORTED_WALLETS[key]
-        if (option.connector === connector) {
-          if (option.connector === injected) {
-            if (isMetamask && option.name !== 'MetaMask') {
-              return null
-            }
-            if (!isMetamask && option.name === 'MetaMask') {
-              return null
-            }
-          }
-          return (
-            <Option
-              id={`connect-${key}`}
-              key={key}
-              clickable={false}
-              color={option.color}
-              header={option.name}
-              subheader={option.description}
-              icon={require(`../../assets/images/${isDarkMode ? '' : 'light-'}${option.iconName}`)}
-            />
-          )
-        }
-        return null
-      })}
+      {isMetamask && isCoin98 && isInjected && <WarningBox option={option} />}
```

Comment:
Why operator `&&` here between coin98 and metamask ?

Reply:
check if both is installed and avoid connector !== injected